### ISSUE TITLE
Ensure That There's Some Way of Running Python 2.7 (and other versions) from the GH Interface

### ISSUE THREAD

**Issue by chriswmackey** [ladybug-analysis-tools/ladybug#18]
The method described in this  email with @mostaphaRoudsari and @Vyrwn sounds good to me:

I have been thinking about the best solution for integrating scripts such as yours (@Vyrwn) in ladybug, honeybee and now butterfly environment. Both Grasshopper and IronPython have their own limitations over using Python.

My first thought is to collect them in a folder named as scripts and then load them using a dynamic component that recreates itself based on the arguments and returns of the script. So we can call them using python and load the results back into Grasshopper.

**Comment by didimitrie** [comment]
I'm willing to help out on this. I've prototyped a [workflow for cloud components](https://www.youtube.com/watch?v=tT-wYlM0Aqg&list=PLlsa4iFBVezNFw25rIOlwaQS4RT6Zz_sh&index=1) [youtube]. The stack seen there is using node-based, but there's nothing stopping from implementing/calling any script from the so called `exec` function, ie python scripts, ruby scripts, other js stuff, even .net stuff. 

Of course, the component server can run locally (localhost) or wherever else, added scalability advantages :)

**Comment by didimitrie** [comment]
Yes, re timeline: it's a mess 💥  All these things are coming out from my research project at the bartlett/innochain and I'm finding it difficult to invest time in them (ie, justify to the board) to be able to gear up to a proper minimal-ish release. 

I'll see if I can speed this up a bit. Let me know your timeline as well (if there is one), it might help shape mine.
